doc/add-ons.md

```diff
@@ -2,6 +2,8 @@
 
 Hubs client add-ons are pluggable libraries that are installed as part of the client and loaded at runtime. Add-ons allow functionality to the Hubs core and allow easy Hubs client extensibility while maintaining the Hubs client core lean and minimal.
 
+The currently installed addons are listed in the `addons` array inside the `addons.json` file at the root of the Hubs client source. Several are included with the `addons` code branch.
```

Comment:
What about changing this from "Several are included with the `addons` code branch." to something like "Several are included by default, see `[Currently available add-ons](#currently-available-add-ons)`."

I think it would be nice to know which ones are included by default (although that section isn't entirely accurate the way it is since the template add-on isn't installed by default, but it could be moved to a subsection), also the addons branch is only temporary, so this would future proof things.

Reply:
done